scripts/install.sh

```
@@ -21,8 +21,35 @@ DATE=$(date +"%Y%m%d-%H%M%S")
+
+# user selection
+DEFAULT_USER="${SUDO_USER:-$USER}"
+
+read -r -p "Which user to use? (Enter for ${DEFAULT_USER}, or type 0 to list users) " OPTION_USER
+
+if [[ -z "$OPTION_USER" ]]; then # if it's empty
+    echo -e "Default user selected..."
+    CURRENT_USER="$DEFAULT_USER"
+elif [[ "$OPTION_USER" == "0" ]]; then # listing all the users
+    echo "Here is the list of non-system users:"
+<<<<<<< HEAD
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)